VocaDb.ResXFileCodeGenerator/StringBuilderGenerator.cs

```diff
@@ -224,14 +237,44 @@ private static IEnumerable<(string key, string value, IXmlLineInfo line)>? ReadR
 		builder.Append(indent);
 		builder.AppendLineLF("/// </summary>");
 
-		builder.Append(indent);
-		builder.Append("public ");
-		builder.Append(options.StaticMembers ? "static " : string.Empty);
-		builder.Append("string");
-		builder.Append(options.NullForgivingOperators ? null : "?");
-		builder.Append(" ");
-		builder.Append(memberName);
-		return true;
+		var numParams = neutralValue.Count(c => c == '{');
+
+		if (numParams == 0)
+		{
+			builder.Append(indent);
+			builder.Append("public ");
+			builder.Append(options.StaticMembers ? "static " : string.Empty);
+			builder.Append("string");
+			builder.Append(options.NullForgivingOperators ? null : "?");
+			builder.Append(" ");
+			builder.Append(memberName);
+			return true;
+		}
+		else
+		{
+			builder.Append(indent);
+			builder.Append("public ");
+			builder.Append(options.StaticMembers ? "static " : string.Empty);
+			builder.Append("string");
+			builder.Append(options.NullForgivingOperators ? null : "?");
+			builder.Append(" ");
+			builder.Append(memberName);
```

Comment:
Is it possible to move these lines out of the if and else blocks, or extract to a method?